api/envoy/api/v3alpha/lds.proto

```diff
@@ -223,4 +223,12 @@ message Listener {
   // If no configuration is specified, Envoy will not attempt to balance active connections between
   // worker threads.
   ConnectionBalanceConfig connection_balance_config = 20;
+
+  // When this flag is set to true, listeners set the *SO_REUSEPORT* socket option and
+  // create one socket for each worker thread. This makes inbound connections
+  // distribute among worker threads roughly evenly in cases where there are a high number
+  // of connections. When this flag is set to false, all worker threads share one socket.
+  // For UDP this flag is set to true forcibly.
+  // Before Linux v4.19-rc1, new TCP connections may be rejected during hot restart.
```

Comment:
Might as well ref link to the patch as well as the original issue? Seems useful.

Reply:
done